storm-core/src/clj/backtype/storm/daemon/nimbus.clj

```
@@ -950,20 +1047,33 @@
             TOPOLOGY-EVENTLOGGER-EXECUTORS (total-conf TOPOLOGY-EVENTLOGGER-EXECUTORS)
             TOPOLOGY-MAX-TASK-PARALLELISM (total-conf TOPOLOGY-MAX-TASK-PARALLELISM)})))
 
+(defn blob-rm-key [blob-store key storm-cluster-state]
+  (try
+    (.deleteBlob blob-store key (get-nimbus-subject))
+    (if (instance? LocalFsBlobStore blob-store)
+      (.remove-blobstore-key! storm-cluster-state key))
+    (catch Exception e)))
```

should not ignore exception. At least log it.